met-web/src/components/survey/building/index.tsx

```diff
@@ -219,11 +220,42 @@ const SurveyFormBuilder = () => {
                             <Switch
                                 checked={formDefinition.display === 'wizard'}
                                 onChange={(e) => {
-                                    if (e.target.checked) {
-                                        setFormDefinition({ display: 'wizard', components: [] });
-                                        return;
-                                    }
-                                    setFormDefinition({ display: 'form', components: [] });
+                                    dispatch(
+                                        openNotificationModal({
+                                            open: true,
+                                            data: {
+                                                header: 'Change Survey Type',
+                                                subText: [
+                                                    {
+                                                        text: `You will be changing the survey type from ${
+                                                            formDefinition.display === 'wizard'
+                                                                ? 'multi page'
+                                                                : 'single page'
+                                                        } to ${
+                                                            formDefinition.display === 'wizard'
```

Comment:
both conditions will always have the same value, just use one condition and full text instead of checking it twice.

Also, make this a boolean flag instead